service/controller/app/v1/key/key.go

```diff
@@ -66,7 +66,7 @@ func InCluster(customResource v1alpha1.App) bool {
 }
 
 func KubeConfigFinalizer(customResource v1alpha1.App) string {
-	return fmt.Sprintf("app-operator.giantswarm.io/%s", customResource.GetName())
+	return fmt.Sprintf("app-operator.giantswarm.io/app-%s", customResource.GetName())
```

Comment:
So how to deal with the differently named finalizers we already have?

Reply:
I don't think we use this in production. Only for testing in some test installations.

I requested the review from @giantswarm/team-batman to confirm.

Reply:
Yes, we only have few cases for testing. We can go with it.

Reply:
EFK is running in prod in one CP but we're at an early stage. So I think we can change this and good to get it straight.